jsapp/js/components/common/koboTagsInput.tsx

```diff
@@ -80,7 +80,6 @@ class KoboTagsInput extends React.Component<KoboTagsInputProps> {
           inputProps={inputProps}
           onlyUnique
           addOnBlur
-          addOnPaste
```

Comment:
Ok, note here that you are modifying `KoboTagsInput`. This component is a wrapper for `react-tagsinput`, and we use it in most of the places we render tags editor. But in NLP we don't use it, we use `react-tagsinput` directly (so that we can customize it more). The component you should look at is `TagsResponseForm`. And there is already [a PR](https://github.com/kobotoolbox/kpi/pull/4905/files) that modifies it - I've reassigned it to you right now :)

Reply:
This way we fix both problems:
- pasting doesn't split by spaces
- pasting doesn't remove the already typed in text

Reply:
Sounds good!